docs/utilities.md

```diff
@@ -611,10 +611,12 @@ process will get the schema and replay the log file from the remote
 source (e.g. in the case of tickerplant subscriptions).
 
 .sub.getsubscriptionhandles is used to get a table of processes to
-subscribe to. The following can be used to return a table of all
-connected processes of type tickerplant:
+subscribe to. It takes a process type and process name, and `()` and a null symbol can be used for all:
```

Comment:
I think this needs slightly reworded + line wrapped at 80 chars because this is a markdown file:
```suggestion
subscribe to. It takes a process type and process name, where `()` or a null
symbol can be used for all:
```